packages/expo-image-manipulator/src/index.ts

```diff
@@ -0,0 +1,53 @@
+import { NativeModulesProxy } from 'expo-core';
+
+const { ExpoImageManipulator } = NativeModulesProxy;
```

Comment:
This will remove support for `web`, please add `ExpoImageManipulator{.web,}.ts` imports